src/main/kotlin/com/cognifide/gradle/aem/tooling/tasks/Sync.kt

```diff
@@ -102,12 +103,13 @@ open class Sync : AemDefaultTask() {
             }
         }
     }
+
     private fun prepareContent() {
         logger.info("Preparing files to be cleaned up (before copying new ones) using filter: $filter")
 
         filterRootDirs.forEach { root ->
             logger.lifecycle("Preparing root: $root")
-            cleaner.prepare(root)
+            cleaner.prepare(Root.normalize(root))
```

Comment:
Maybe move methods from root into vlt filter.filterRootDirs

Reply:
Ensure uniqueness (because of cropping JCR content), rename dirs - >files

Reply:
it would be a error when someone try to sync from /path/jcr:conten/subpath1 and /path/jcr:conten/subpath2 in the same time